cogs/sniper.py

```diff
@@ -123,15 +126,22 @@ async def attempt_snipe(channel, message_cache, message_cache_lock, success_call
                 cache_index += 1
 
         if sniped_user and sniped_content:
+            self.last_snipe = datetime.now()
             await success_callback(channel, sniped_user, sniped_content, sniped_extras, sniped_at)
         else:
-            await Sniper.send_failure_response(channel)
+            await Sniper.send_failure_response(self, message, channel)
 
     @staticmethod
-    async def send_failure_response(channel):
-        embed = create_basic_embed(TEXT_SNIPER_FAIL)
-        embed.set_thumbnail(url=URL_SWIPER_ICON)
-        await channel.send(embed=embed)
+    async def send_failure_response(self, message, channel):
+        time_delta = (datetime.now() - self.last_snipe).total_seconds()
```

Comment:
this method should be static - i.e. shouldn't hold a reference to `self`. a cleaner way to handle this would be to compute the time delta in a non-static method and pass the result to this one.